internal/provider/resource_globalaccount_role_collection_test.go

```diff
@@ -51,6 +80,6 @@ func hclResourceGlobalAccountRoleCollection(resourceName string, displayName str
 	return fmt.Sprintf(`resource "btp_globalaccount_role_collection" "%s" {
         name         = "%s"
         description  = "%s"
-	roles        = %v
+	    roles        = %v
```

Comment:
Could you please run `go fmt ./...` again? This doesn't look properly formatted to me.

Reply:
Fixed, it was tab vs. spaces

Reply:
go fmt does not resolve that however